src/main/groovy/org/fundacionjala/gradle/plugins/enforce/utils/salesforce/component/validators/files/DashboardSalesforceValidator.groovy

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.gradle.plugins.enforce.utils.salesforce.component.validators.files
+
+import org.fundacionjala.gradle.plugins.enforce.utils.Constants
+import org.fundacionjala.gradle.plugins.enforce.utils.Util
+import org.fundacionjala.gradle.plugins.enforce.utils.salesforce.MetadataComponents
+
+
+class DashboardSalesforceValidator implements SalesforceValidator {
+
+    /**
+     * Validates the report file that belongs to dashboards folder defined in Salesforce
+     * @param file is File that will be validate
+     * @param folderComponent is a String
+     * @return true if file is valid
+     */
+    @Override
+    boolean validateFile(File file, String folderComponent) {
+        boolean isValidExtension = Util.getFileExtension(file).equals(MetadataComponents.DASHBOARDS.getExtension()) ||
+                file.getName().endsWith(Constants.META_XML)
+        return folderComponent.equals(MetadataComponents.DASHBOARDS.getDirectory()) &&
```

Comment:
did you think evaluate isValidExtension first?? if it is false, we don't need validate the others